CATS/Problem/Parser.pm

```diff
@@ -283,7 +267,12 @@ sub on_end_tag {
     $h->{e}->($self, \%atts, $el) if $h && $h->{e};
     if (my $stml = $self->current_tag->{stml}) {
         $$stml .= "</$el>" if $el ne 'include';
-        return;
+        if ($h && $h->{in_stml}) {
+            ${$self->{tag_stack}->[-2]->{stml}} .= $$stml ;
+        }
+        else {
+            return;
```

Comment:
Style: Invert condition to avoid else-return. However, this code is perhaps wrong anyway, see below.